### ISSUE TITLE
[BUG] _updateAvailableDevices continuously throws uncaught rejection

### ISSUE THREAD

**Issue by CoreyCole** [twilio/twilio-client.js#124]
- [X] I have verified that the issue occurs with the latest twilio.js release and is not marked as a known issue in the [CHANGELOG.md](https://github.com/twilio/twilio-client.js/blob/master/CHANGELOG.md).
 - [X] I reviewed the [Common Issues](https://github.com/twilio/twilio-client.js/blob/master/COMMON_ISSUES.md) and open GitHub issues and verified that this report represents a potentially new issue.
 - [X] I verified that the [Quickstart](https://github.com/TwilioDevEd/client-quickstart-js) application works in my environment.
 - [X] I am not sharing any [Personally Identifiable Information (PII)](https://www.twilio.com/docs/glossary/what-is-personally-identifiable-information-pii)
       or sensitive account information (API keys, credentials, etc.) when reporting this issue.

**Code to reproduce the issue:**
connect a call on mobile android Google Chrome, see error:
```
// "This browser does not support audio output selection"
// from OutputDeviceCollection.prototype.set
if (!this._isSupported) {
  return Promise.reject(
    new errors_1.NotSupportedError("This browser does not support audio output selection"),
  );
}
```

 I was able to stop the errors by adding a check `outputDevices._isSupported` in the `_updateAvailableDevices` body:
```javascript
_this._updateAvailableDevices = function() {
  if (!_this._mediaDevices) {
    return Promise.reject("Enumeration not supported");
  }
  return _this._mediaDevices.enumerateDevices().then(function(devices) {
    _this._updateDevices(
      devices.filter(function(d) {
        return d.kind === "audiooutput";
      }),
      _this.availableOutputDevices,
      _this._removeLostOutput,
    );
    _this._updateDevices(
      devices.filter(function(d) {
        return d.kind === "audioinput";
      }),
      _this.availableInputDevices,
      _this._removeLostInput,
    );
    var defaultDevice =
      _this.availableOutputDevices.get("default") || Array.from(_this.availableOutputDevices.values())[0];
    [_this.speakerDevices, _this.ringtoneDevices].forEach(function(outputDevices) {
      // I added isSupported check here
      if (outputDevices._isSupported && !outputDevices.get().size && _this.availableOutputDevices.size) {
         // I added the catch here as well, but now it is not called so it throws no error
        outputDevices.set(defaultDevice.deviceId).catch(err => console.error(err));  
      }
    });
  });
};
```
But I don't know if this will have side effects. Could anyone advise? 

**Software versions:**

 - [x] Browser(s): mobile google chrome 80.0.3987.99
 - [x] Operating System: android 
 - [x] twilio.js: 1.10
 - [x] Third-party libraries (e.g., Angular, React, etc.): vanilla

**Comment by charliesantos** [comment]
Hi @CoreyCole ,

Thank you for reporting. Is this happening only on mobile google chrome or are you also able to reproduce on other PC browsers?

**Comment by CoreyCole** [comment]
Hey Charlie, thank you for your response. I only saw this on mobile google chrome on android

**Comment by charliesantos** [comment]
Thanks @CoreyCole . Our team will investigate this issue.

**Comment by charliesantos** [comment]
Create Jira [CLIENT-7373](https://issues.corp.twilio.com/browse/CLIENT-7373)

**Comment by charliesantos** [comment]
Fixed in 1.10.1
